arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/EvalOnTests.kt

```diff
@@ -36,7 +37,7 @@ class EvalOnTests : ArrowFxSpec(spec = {
 
   "evalOn on the same context doesn't dispatch" {
     suspend fun onSameContext(): String =
-      evalOn(ComputationPool) {
+      withContext(ComputationPool) {
```

Comment:
This is just testing code from another library now.
If we want to make such a change we instead just remove `EvalOnTests` from `Arrow Fx Coroutines` since `evalOn` now only delegates to `withContext`.

Reply:
Ok I will do that